### PR TITLE
pythonPackages.pyomo: init at 5.6.5

### PR DESCRIPTION
**Pull request by costrouc**
Closes #37372. PR was not being fixed.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by costrouc** [comment]
@GrahamcOfBorg build python2Packages.pyomo python3Packages.pyomo

**Comment by Lassulus** [comment]
I will merge this tomorrow as https://github.com/NixOS/nixpkgs/pull/37372 is no longer up2date.

File: pkgs/development/python-modules/pyomo/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  ];
+
+  checkPhase = ''
+    rm pyomo/bilevel/tests/test_blp.py \
+       pyomo/version/tests/test_installer.py
+    nosetests
+  '';
+
+  meta = with lib; {
+    description = "Pyomo: Python Optimization Modeling Objects";
```
Comment (dotlambda) [inline pkgs/development/python-modules/pyomo/default.nix]: The package name should not be included in the description: https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes

Comment (costrouc) [reply pkgs/development/python-modules/pyomo/default.nix]: This PR looks like it was already merged. I can still make the quick fix if needed.

Comment (dotlambda) [reply pkgs/development/python-modules/pyomo/default.nix]: Not needed, but please adhere to the contributing guidelines next time!

**Pull request merged**